src/main/resources/anet-schema.yml

```diff
@@ -534,3 +534,12 @@ properties:
             options:
               type: object
               additionalProperties: true
+
+  automaticallyInactivateUsers:
```

Comment:
Please provide a title and a description for the new keys.

Reply:
Sure!